### PR TITLE
ACMS-1435: Enable Github actions.

### PR DESCRIPTION
**Pull request by vishalkhode1**
**Motivation**
ACMS-1435: Enable Github actions.

**Proposed changes**
Migrate the existing push/pr travis CI jobs to Github actions.

**Alternatives considered**
N/A

**Testing steps**
Github actions CI job should pass.

**Merge requirements**
- [ ] _Major change_
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: .github/workflows/acquia_cms_ci.yml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      SITESTUDIO_API_KEY: ${{ secrets.SITESTUDIO_API_KEY }}
+      SITESTUDIO_ORG_KEY: ${{ secrets.SITESTUDIO_ORG_KEY }}
+      ACMS_JOB: "base"
+    strategy:
+      matrix:
+        orca-job:
+          - ISOLATED_TEST_ON_CURRENT_DEV
+        #php-version: [ "7.4" ]
+        module:
+          - acquia_cms_article
```
Comment (rajeshreeputra) [inline .github/workflows/acquia_cms_ci.yml:101]: For integrated test acquia_cms need to be added in the list

**Review by chandan-singh7929 — APPROVED** [review]
@vishalkhode1 - Overall looks good, though one observation, I don't see test with php 8.x can we add some?

**Review by deepakmishra2 — APPROVED** [review]
LGTM

**Review by rajeshreeputra — APPROVED** [review]
Looks good to me. This is great start!!!
We should add test with PHP 8.1 agree with @chandan-singh7929.

**Review by apathak18 — COMMENTED** [review]
All the changes made with this PR ~~ LGTM 
+1 with @chandan-singh7929 suggestion, we do need a PHP8.1 support with this github actions.
Also I'm seeing few of the keys are commented is it done on purpose? @vishalkhode1

**Review by panshulK — APPROVED** [review]

**Comment by vipin-mittal-acquia** [comment]
@vishalkhode1 : Remove .travis.yml file completely

**Comment by vishalkhode1** [comment]
> @vishalkhode1 : Remove .travis.yml file completely

We can't remove that right now. As currently., the CRON CI jobs are running on Travis CI only. Once this ticket is completed (i.e migrating push/branch CI jobs from Travis CI to Github actions), then I'll work migrating CRON CI jobs from Travis CI to Github Actions and will remove all files related to Travis CI.

**Pull request merged**